src/run.ts

```diff
@@ -278,22 +284,33 @@ ${
 
   let searchResult = await searchResultPromise;
   console.log(JSON.stringify(searchResult.data, null, 2));
+
+  let messageBody = await prBodyPromise;
```

Comment:
Since we have `prBodyPromise`, this variable probably should be called `prBody` to make the connection between them obvious.